modules/charon-core/src/test/java/org/wso2/charon3/core/utils/PatchOperationUtilTest.java

```diff
@@ -0,0 +1,400 @@
+                        READ_WRITE, DEFAULT, NONE, null, null, null);
+        AttributeSchema attributeSchema4 =
+                SCIMAttributeSchema.createSCIMAttributeSchema(
+                        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:manager",
+                        "manager", COMPLEX, false, "", false, false,
+                        READ_WRITE, DEFAULT, NONE, null, null, subAttributeSchemaList2);
+        SCIMResourceTypeSchema scimResourceTypeSchema =
+                SCIMResourceTypeSchema.createSCIMResourceSchema(schemasList, attributeSchema1, attributeSchema2,
+                        attributeSchema3, attributeSchema4);
+
+        return scimResourceTypeSchema;
+    }
+
+    private User getUser() throws InstantiationException, IllegalAccessException {
+
+        User user = User.class.newInstance();
+        user.setSchema("urn:ietf:params:scim:schemas:core:2.0:User");
+        user.setSchema("urn:ietf:params:scim:schemas:extension:enterprise:2.0:User");
+
+        SimpleAttribute simpleAttribute = new SimpleAttribute("nickName", "shaggy");
+        simpleAttribute.setMutability(SCIMDefinitions.Mutability.READ_WRITE);
+        simpleAttribute.setRequired(false);
+        user.setAttribute(simpleAttribute);
+
+        SimpleAttribute simpleAttribute1 = new SimpleAttribute("id",
+                "229d3f0d-a07b-4052-bf4d-3071ecafed04");
+        simpleAttribute1.setMutability(SCIMDefinitions.Mutability.READ_ONLY);
+        simpleAttribute1.setRequired(false);
+        user.setAttribute(simpleAttribute1);
+
+        ComplexAttribute complexAttributeMeta = new ComplexAttribute("meta");
+        user.setAttribute(complexAttributeMeta);
+
+        SimpleAttribute simpleAttributeEmailType = new SimpleAttribute("type", "home");
+        SimpleAttribute simpleAttributeEmailValue = new SimpleAttribute("value", "rash@gmail.com");
+        simpleAttributeEmailValue.setMutability(READ_WRITE);
+        simpleAttributeEmailValue.setMultiValued(false);
+
+        Map<String, Attribute> valuesMap = new HashMap<>();
+        valuesMap.put("type", simpleAttributeEmailType);
+        valuesMap.put("value", simpleAttributeEmailValue);
+
+        ComplexAttribute complexAttributeEmail = new ComplexAttribute("emails_rash@gmail.com_home");
+        complexAttributeEmail.setType(COMPLEX);
+        complexAttributeEmail.setMutability(READ_WRITE);
+        complexAttributeEmail.setRequired(false);
+        complexAttributeEmail.setSubAttributesList(valuesMap);
+
+        MultiValuedAttribute multiValuedAttributeEmail = new MultiValuedAttribute("emails");
+        multiValuedAttributeEmail.setType(SCIMDefinitions.DataType.COMPLEX);
+        multiValuedAttributeEmail.setMultiValued(true);
+        multiValuedAttributeEmail.setMutability(READ_WRITE);
+        multiValuedAttributeEmail.setAttributeValue(complexAttributeEmail);
+        user.setAttribute(multiValuedAttributeEmail);
+
+        return user;
+    }
+
```

Comment:
Unnecessary new line

Reply:
fixed in 6e37f6e580f0e7c973edbc994d55b5a8062d5c14